cmd/glasskube/cmd/list.go

```diff
@@ -129,7 +130,7 @@ func versionString(pkg list.PackageWithStatus) string {
 			if statusVersion != specVersion {
 				versionAddons = append(versionAddons, fmt.Sprintf("%v desired", specVersion))
 			}
-			if repoVersion != "" && statusVersion != repoVersion {
+			if repoVersion != "" && !semver.IsUpgradable(repoVersion, statusVersion) && statusVersion != repoVersion {
```

Comment:
I see two issues here:
1. I think `semver.IsUpgradable(statusVersion, repoVersion)` would make more sense than `!semver.IsUpgradable(repoVersion, statusVersion)`, since we want to know whether the actual version could be upgraded to the one from the repo, not the other way around.
2. `statusVersion != repoVersion` should be omitted, since this would make the condition true, even if IsUpgradable evalutes to false

Reply:
ohk